Sources/Models/Connect/Account.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  Account.swift
+//  Stripe
+//
+//  Created by Andrew Edwards on 7/8/17.
+//
+//
+
+import Foundation
+import Vapor
+import Helpers
+
+/**
+ Account Model
+ https://stripe.com/docs/api#account_object
+ */
+
+public final class ConnectAccount: StripeModelProtocol {
+    
+    public private(set) var id: String?
+    public private(set) var object: String?
+    public private(set) var businessName: String?
+    public private(set) var businessUrl: String?
+    public private(set) var chargesEnabled: Bool?
+    //public private(set) var transfersEnabled: Bool?
+    public private(set) var country: String?
+    public private(set) var debitNegativeBalances: Bool?
+    public private(set) var declineChargeOn: Node?
+    public private(set) var defaultCurrency: StripeCurrency?
+    public private(set) var detailsSubmitted: Bool?
+    public private(set) var displayName: String?
+    public private(set) var email: String?
+    public private(set) var externalAccounts: ExternalAccounts?
+    public private(set) var legalEntity: LegalEntity?
+    public private(set) var metadata: Node?
+    public private(set) var payoutSchedule: PayoutSchedule?
+    public private(set) var payoutStatementDescriptor: String?
+    public private(set) var payoutsEnabled: Bool?
+    public private(set) var productDescription: String?
+    public private(set) var statementDescriptor: String?
+    public private(set) var supportEmail: String?
+    public private(set) var supportPhone: String?
+    public private(set) var timezone: String?
+    public private(set) var tosAcceptance: TOSAcceptance?
+    public private(set) var type: String?
+    public private(set) var verification: Verification?
+    public private(set) var keys: Node?
+    
+    public init(node: Node) throws {
+        self.id = try node.get("id")
+        self.object = try node.get("object")
+        self.businessName = try node.get("business_name")
+        self.businessUrl = try node.get("business_url")
+        self.chargesEnabled = try node.get("charges_enabled")
+        // TODO - investigate, not getting value from response after rejecting account. 
+        // https://stripe.com/docs/api/curl#reject_account
+        // self.transfersEnabled = try node.get("transfers_enabled")
```

Comment:
More of stripe. It's confusing. It studs that I should get this return value when rejecting an account but I never see it.